pkg/wasm/convert.go

```diff
@@ -21,15 +21,48 @@ import (
 	udpa "github.com/cncf/xds/go/udpa/type/v1"
 	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
 	wasm "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/wasm/v3"
+	wasmv3 "github.com/envoyproxy/go-control-plane/envoy/extensions/wasm/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/conversion"
 	"go.uber.org/atomic"
 	anypb "google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 
 	extensions "istio.io/api/extensions/v1alpha1"
 	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pilot/pkg/util/protoconv"
 	"istio.io/istio/pkg/config/xds"
 )
 
+var (
+	allowWasmHTTPFilter = &wasm.Wasm{
+		Config: &wasmv3.PluginConfig{
```

Comment:
does this actually have to be WASM? Since this assumes we have attributegen compiled in which may not be something we have long term

Reply:
I just wanted to use Wasm only in the allow list. (https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/config_source.proto#config-core-v3-extensionconfigsource field type_urls.)

RBAC or some other  filter would be better?

Reply:
BTW, I agree with your point.
Then, I think RBAC would be the one of the candidates. (I don't think RBAC is not included in Envoy).
Any other good candidates?

Reply:
Done to move it to RBAC.
If you have any other idea, please let me know. 
Thanks!

Reply:
Rolled back at this time.